packages/builder/src/pages/builder/portal/apps/onboarding/_components/NamePanel.svelte

```diff
@@ -31,6 +31,18 @@
       return "Invalid URL"
     }
   }
+
+  $: urlManuallySet = false
```

Comment:
This is not related to this ticket, changing the onboarding app to URL sync for an better dev/user experience